### DISCUSSION TITLE
Deep Local Analysis

### DISCUSSION THREAD

**Discussion by erikedlund** [martinez-zacharya/TRILL#15] [Ideas]
Hello and thank you for creating such a cool resource! I'm not sure what sort of process you have in mind for adding models to the zoo but I'd love to see Deep Local Analysis made available, specifically the tool for assessing mutation-induced changes of binding affinity.
http://gitlab.lcqb.upmc.fr/DLA/DLA
https://pubmed.ncbi.nlm.nih.gov/35962985/

**Comment by martinez-zacharya** [comment]
Thank you very much for your interest! I'll definitely add this to my goals for the next major release, thank you for letting me know what might be of use to researchers!
